tests/conftest.py

```diff
@@ -62,6 +63,12 @@ def biosero_auth_headers(app):
         return {"Authorization": app.config.get("API_TOKENS_EVENTS").get("biosero_read_write")}
 
 
+@pytest.fixture
+def beckman_auth_headers(app):
+    with app.app_context():
+        return {"Authorization": app.config.get("API_TOKENS_EVENTS").get("beckman_read_write")}
```

Comment:
I think we dont have api key checking for Beckman endpoints currently so this is not needed